format.sh

```diff
@@ -3,7 +3,7 @@
 FORMAT_VER="11"
 FORMAT_OPTS="-i -style=file"
 TIDY_OPTS="-p . --fix --fix-errors"
-COMPILER_OPTS="-fno-builtin -std=gnu90 -Iinclude -Isrc -D_LANGUAGE_C -DNON_MATCHING"
+COMPILER_OPTS="-fno-builtin -fsyntax-only -std=gnu90 -Iinclude -Isrc -Iassets -Ibuild -I. -D_LANGUAGE_C -DNON_MATCHING"
```

Comment:
```suggestion
COMPILER_OPTS="-fno-builtin -fsyntax-only -std=gnu90 -Iinclude -Isrc -Ibuild -I. -D_LANGUAGE_C -DNON_MATCHING"
```
smh sabotaging your own work

more widely this could be copypasted from `CC_CHECK` in the makefile
https://github.com/zeldaret/oot/blob/6f0b8f1e7483524cc967536c50cb02e431d534c9/Makefile#L133

Reply:
Good catch on the `-Iassets` 👍

If by copypasting from `CC_CHECK` you meant adding the other stuff, I don't think it makes any difference in the context of clang-tidy, since it mostly just needs to be able to preprocess and compile the file afaik

Reply:
oh I thought those were also passed to clang-format, nvm then :sweat_smile: